pkg/iostreams/iostreams.go

```diff
@@ -289,13 +294,22 @@ func System() *IOStreams {
 	stdoutIsTTY := isTerminal(os.Stdout)
 	stderrIsTTY := isTerminal(os.Stderr)
 
+	hasTrueColor := false
+	if stdoutIsTTY {
+		// also enables truecolor support for NewColorable below
+		if err := enableVirtualTerminalProcessing(os.Stdout); err == nil {
+			hasTrueColor = true
```

Comment:
This looks like it unconditionally considers that every non-Windows terminal supports truecolor. We will rely on TERM and COLORTERM variables instead. I'll push an update